src/Compilers/Shared/CoreClrBuildClient.cs

```diff
@@ -25,6 +26,9 @@ private CoreClrBuildClient(RequestLanguage language, CompileFunc compileFunc)
 
         internal static int Run(IEnumerable<string> arguments, RequestLanguage language, CompileFunc compileFunc)
         {
+            // Register encodings for console
+            Encoding.RegisterProvider(CodePagesEncodingProvider.Instance);
```

Comment:
I'm not sure. I don't think so, but I also would rather not take that risk in ask mode.